### DISCUSSION TITLE
Best way to access the registry

### DISCUSSION THREAD

**Discussion by bartbutenaers** [QuorumDMS/ftp-srv#330] [Q&A] [answered]
Hi everybody,

I would like to show in my user interface a list of all supported commands, and their description.
All the info that I need is available in the [registry](https://github.com/QuorumDMS/ftp-srv/blob/main/src/commands/registry.js) already.
I am wondering what is the best way to access that registry from my code?
At first sight I don't think I can access it from my FtpSrv server instance, but I can be mistaken...

Thanks!!!
Bart

**Comment by matt-forster** [comment, marked as answer]
Good question - It isn't possible through the interface at the moment. I would totally be open to exposing the _list_ of commands (without the functions) in a get function or similar in the interface to expose this information.

**Comment by bartbutenaers** [comment]
@matt-forster,
I made the changes locally on my pc to implement this, and  it seems to work fine.  But I will wait to create a pull request, until my TLS renewal PR has been merged.  Because I have been so smart to do the TLS renewal section in my master branch on Github, so things would get mixed up when I create a PR now...
